cadc-vos-server/src/main/java/org/opencadc/vospace/server/actions/CreateNodeAction.java

```diff
@@ -199,7 +199,7 @@ public void doAction() throws Exception {
         
         // output modified node
         NodeWriter nodeWriter = getNodeWriter();
-        syncOutput.setCode(200); // Created??
+        syncOutput.setCode(201); // Created??
```

Comment:
comment should be removed since it was there to note this exact thing (201 means "Created" in HTTP)

Reply:
Removed the comment.